src/cart/crt_iv.c

```diff
@@ -1952,34 +2113,35 @@ call_pre_sync_cb(struct crt_ivns_internal *ivns_internal,
 	}
 	need_put = true;
 
-	if (rpc_req->cr_co_bulk_hdl != CRT_BULK_NULL) {
-		D_ALLOC_ARRAY(tmp_iovs, iv_value.sg_nr);
-		if (tmp_iovs == NULL) {
-			D_ERROR("Failed to allocate temporary iovs\n");
-			D_GOTO(exit, rc);
-		}
+	D_ALLOC_ARRAY(tmp_iovs, iv_value.sg_nr);
```

Comment:
why is check for null bulk handle removed?

Reply:
Good catch.  Shouldn't have been eliminated.  Back in place.